src/components/ActivityDetails.js

```diff
@@ -0,0 +1,190 @@
+            aria-label="connection status" />
+        { text || 'Failed' }
+    </Text>
+})[status];
+
+const statusText = (executorStatus) => ({
+    running: <Text component={ TextVariants.p } >
+        Running
+    </Text >,
+    success: <Text className="ins-c-remediations-success" component={ TextVariants.p }>
+        Suceeded
+    </Text>,
+    failure: <Text className="ins-c-remediations-failure" component={ TextVariants.p }>
+        Failed
+    </Text>
+})[executorStatus];
+
+const statusSummary = (executorStatus, systemsStatus) => {
+    console.log(executorStatus, systemsStatus);
+    return <Split style={ { display: 'flex' } } className="ins-c-remediations-status-bar">
+        <SplitItem>
+            { statusText('success') }
+        </SplitItem>
+        <SplitItem>
+            { renderStatus('success', systemsStatus.success) }
+        </SplitItem>
+        <SplitItem>
+            { renderStatus('failure', systemsStatus.failure) }
+        </SplitItem>
+        <SplitItem>
+            { renderStatus('running', systemsStatus.running) }
+        </SplitItem>
+    </Split>;
+};
+
+const ActivityDetail = ({
+    remediation,
+    playbookRun,
+    playbookRunSystems,
+    getPlaybookRun,
+    getPlaybookRunSystems
+}) => {
+    useEffect(() => {
+        getPlaybookRun();
+        getPlaybookRunSystems();
+
+    }, []);
+
+    // const systemsStatus = playbookRunSystems.reduce((acc, { status }) => ({ ...acc, [normalizeStatus(status)]: acc[normalizeStatus(status)] + 1 })
+    //     , {  running: 0, success: 0, failure: 0 });
+    const systemsStatus = { running: 1, success: 2, failure: 1 };
+
+    return (
+        <React.Fragment>
+            <PageHeader>
+                <Breadcrumb>
+                    <BreadcrumbItem>
+                        <Link to={ `/${remediation.id}` }> { remediation.name } </Link>
+                    </BreadcrumbItem>
+                    <BreadcrumbItem isActive> <DateFormat type='exact' date={ playbookRun.data.created_at } /> </BreadcrumbItem>
+                </Breadcrumb>
+                <Stack gutter>
+                    <StackItem>
+                        <PageHeaderTitle title={ <DateFormat type='exact' date={ playbookRun.data.created_at } /> } />
+                    </StackItem>
+                    <StackItem>
+                        <Split gutter>
+                            <SplitItem>
+                                <DescriptionList className='ins-c-playbookSummary__settings' title='Run on'>
+                                    <DateFormat type='exact' date={ playbookRun.data.created_at } />
+                                </DescriptionList>
+                            </SplitItem>
+                            <SplitItem>
+                                <DescriptionList className='ins-c-playbookSummary__settings' title='Run by'>
+                                    { `${playbookRun.data.created_by.first_name} ${playbookRun.data.created_by.last_name}` }
```

Comment:
https://github.com/RedHatInsights/insights-remediations-frontend/blob/35a1476d6bea26cea69f2cd856ae6a6ceb52345c/src/Utilities/model.js#L32 can be reused here